firestore.rules

```diff
@@ -38,5 +38,9 @@ service cloud.firestore {
       allow create: if request.auth.token.email in get(/databases/$(database)/documents/samwise-groups/$(request.resource.data.group)).data.members 
       allow read, delete: if request.auth.token.email == resource.data.invitee
     }
+    match /samwise-groups/{group} {
+      allow create: if canCreate()
+      allow read, delete: if request.auth.token.email in get(/databases/$(database)/documents/samwise-groups/$(request.resource.data.group)).data.members 
```

Comment:
Use `resource` instead of `request.resource`
- `request.resource` is your edits (i.e. the stuff you pass in `xxxCollection.doc(...).update()`)
- `resource` is the stuff already in the db.